Doppler.BigQueryMicroservice/Entitites/EmailSender/RelayEmailSenderConfiguration.cs

```diff
@@ -18,5 +18,6 @@ public class RelayEmailSenderConfiguration
         public string Username { get; set; }
         public string FromName { get; set; }
         public string FromAddress { get; set; }
+        public Dictionary<string, string> ReplyTo { get; set; }
```

Comment:
Mmm, yo usaría un approach parecido al de From para ser consistente:

```suggestion
        public string ReplyToName { get; set; }
        public string ReplyToAddress { get; set; }
```

Además de evitar nombres dinámicos queue son propensos a typos.

Reply:
ok, lo habia hecho guiandome de la configuracion de los templates, como para que quede consistente el archivo, si quieres me regalas un par de minutos y lo checamos juntos

Reply:
Creo que en la de templates aplica el diccionario porque eventualmente podríamos tener más lenguajes. En cambio en este caso `ReplyToName` y `ReplyToAddress` serían las únicas y los nombres son arbitrarios.

Reply:
sugerencia aceptada y aplicada